source/quickguides/quick-guide-generic.rst

```diff
@@ -0,0 +1,248 @@
+        runner.run_diags([param])
+
+
+Run in serial with:
+
+    ::
+
+        python run_e3sm_diags.py
+
+The above run has the same results as :ref:`the parameter file linked here <cori-params-v1>`,
+which was run using ``e3sm_diags -p lat_lon_demo.py``.
+This new way of running (no ``-p``) is implemented in version 2.0.0,
+preparing ``e3sm_diags`` to accomodate more diagnostics sets with set-specific parameters.
+
+
+#expand multiprocessing#
+
+View results on the web
+'''''''''''''''''''''''
+Once the run is completed,
+open  ``#expand web_address#lat_lon_demo/viewer/index.html`` to view the results.
+
+**Tip:** Once you're on the webpage for a specific plot, click on the
+'Output Metadata' drop down menu to view the metadata for the displayed plot.
+Running that command allows the displayed plot to be recreated.
+Changing any of the options will modify the just that resulting figure.
+
+
+
+Running all the core diagnostics sets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Core diagnostics set includes:
+**lat_lon**, **zonal_mean_xy**, **zonal_mean_2d**, **polar**, **cosp_histogram**,
+**meridional_mean_2d**.
+These diags share a common parameter space (core parameters).
+To run all these sets without defining set-specific parameters
+(e.g. **plev** for **zonal_mean_2d** and **meridional_mean_2d**.),
+replace the ``runner.sets_to_run`` line in ``run_e3sm_diags.py`` with the one below:
+
+ ::
+
+   runner.sets_to_run = ['lat_lon','zonal_mean_xy', 'zonal_mean_2d', 'polar', 'cosp_histogram', 'meridional_mean_2d']
+
+
+Running area mean time series set
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In v2.0.0, the time series set was implemented to support regional averaged time series plot
+using monthly mean time series input.
+This set is enabled if monthly mean time series is processed as documented
+:doc:`here <../input-data-requirement>`.
+
+A ``run_e3sm_diags.py`` example for running area mean time series alone:
+
+    .. code:: python
+
+        import os
+        from acme_diags.parameter.core_parameter import CoreParameter
+        from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+        from acme_diags.run import runner
+        
+        param = CoreParameter()
+        
+        param.reference_data_path = '#expand obs_path#time-series/'
+        param.test_data_path = '#expand test_data_path#time-series/E3SM_v1/'
+        param.test_name = 'e3sm_v1'
+        
+        prefix = '#expand web_address#doc_examples/'
+        param.results_dir = os.path.join(prefix, 'area_mean_with_obs')
+        #param.multiprocessing = True
+        #param.num_workers =  40
+        
+        # We're passing in this new object as well, in
+        # addition to the CoreParameter object.
+        
+        ts_param = AreaMeanTimeSeriesParameter()
+        #ts_param.ref_names = ['none']   # This setting plot model data only
```

Comment:
What does "This setting plot model data only" mean?

- "Only use this setting if you're plotting model data"?
- "Using this setting will plot only the model data, not the observation data"?
- Something else?

Reply:
"Using this setting will plot only the model data, not the observation data" is cleaner